### PR TITLE
Arc 7

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 7/07_00_ARC07_1B.txt [hunk 1]
```diff
@@ -167,13 +167,13 @@ With a suggestive chuckle, Noel-sensei says something rather out of character fo
 -- Page 52, Offset 16702.
 -- ……それは助かる。
 // Better way to phrase the 助かる -> 助かった here?
-...That helped me out.
+She really did me a favor...
```
Comment (Petrikow) [inline script/Arcueid/Day 7/07_00_ARC07_1B.txt:170]: Still think this feels a bit weird. Doesn't do much to fix it

File: script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt [hunk 2]
```diff
@@ -895,8 +895,8 @@ One in ten thousand corpses will survive against the odds, mature,
 [sha:eab29d7480ff0e9635afe024d5a92cc704b10433]{
 -- Page 572, Offset 18008.
 -- やがて王を墜とすように――――彼らもそういうゲームでもしなければ存在してられないの。
-From there, they strive to someday dethrone the king―――
-Without engaging in such games, they wouldn't be able to go on living.
+From there, they strive to someday dethrone the king――
+without engaging in such games, they wouldn't be able to go on living.
```
Comment (Petrikow) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt:899]: the ―― here just feels like an awkward artifact from Japanese. 
Probably a period.

**Review by Petrikow — COMMENTED** [review]
dsa

File: script/Arcueid/Day 7/07_00_ARC07_1B.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_1B.txt:170]: Any suggestions?

Comment (Avuxo) [reply script/Arcueid/Day 7/07_00_ARC07_1B.txt:170]: it might not fix the か->かった but i definitely prefer this wording.

**Pull request merged**